index.bs

```diff
@@ -6152,37 +6152,40 @@ leakage due to such an attack:
     - When verifying an {{AuthenticatorAssertionResponse}} response from the [=authenticator=], make it indistinguishable whether
           verification failed because the signature is invalid or because no such user or credential is registered.
 
-    - Perform a different authentication step, such as username and password authentication,
-        before initiating the WebAuthn [=authentication ceremony=].
-        This moves the username enumeration problem from the WebAuthn [=authentication ceremony=]
+    - Perform a multi-step [=authentication ceremony=], e.g., beginning with supplying username and password or a session cookie,
```

Comment:
Why? Actually, why not we combine this whole argument to, During authentication, whether or not account exists or not, RP should try to get the assertion from the signature.

Reply:
Also, I am not sure about username recommendation we give in this section which says for one time OTP and other things.. Probably a good idea to review username enumeration issue in another PR??

Reply:
>During authentication, whether or not account exists or not, RP should try to get the assertion from the signature.

I don't understand what you mean, could you elaborate on this?

>Also, I am not sure about username recommendation we give in this section which says for one time OTP and other things.. Probably a good idea to review username enumeration issue in another PR??

Yeah, I'm also a bit ambivalent on it. You're welcome to open a new issue with any comments you have.

Reply:
> I don't understand what you mean, could you elaborate on this?

Actually, what I meant was already present in first option in authentication ceremonies.

Reply:
> Yeah, I'm also a bit ambivalent on it. You're welcome to open a new issue with any comments you have.

I will look and open up clarification on the second point later. Thanks.